libs/pxweb2-ui/src/lib/components/Spinner/Spinner.tsx

```diff
@@ -0,0 +1,244 @@
+import cl from 'clsx';
+import classes from './Spinner.module.scss';
+
+export interface SpinnerProps {
+  size?: 'xsmall' | 'small' | 'medium' | 'large' | 'xlarge';
+  variant?: 'default' | 'inverted';
+  label?: string;
+}
+
+export function Spinner({
+  size = 'medium',
+  variant = 'default',
+  label,
+}: SpinnerProps) {
+  let PathStrokeSubtle = '';
+  let PathStrokeAction = '';
+  let Eclipse = null;
+  let Labelsize: 'small' | 'medium' = 'medium';
+
+  switch (variant) {
```

Comment:
This switch and the variables on line 15 and 16 seem to be ununsed. Please remove them.